.github/workflows/windows-packages.yaml

```diff
@@ -0,0 +1,70 @@
+# GitHub Actions configuration for TimescaleDB
+name: Windows Packages
+on:
+  schedule:
+    # run daily 0:00 on master branch
+    - cron: '0 0 * * *'
+  push:
+    tags:
+    - '*'
+    branches:
+    - release_test
+jobs:
+  build:
+    name: Windows package PG${{ matrix.pg }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        pg: [ 12, 13 ]
+        os: [ windows-2019 ]
+    env:
+      # PostgreSQL configuration
+      PGPORT: 6543
+      PGDATA: pgdata
+    steps:
+
+    - name: Checkout TimescaleDB source
+      uses: actions/checkout@v2
+
+    - name: Get version
+      id: version
+      run: |
+        # version will only be a proper version in a release branch so we use update_from_version
+        # as fallback for master
+        if (grep '^version = [0-9.]\+$' version.config)
+        {
+          $version=grep '^version = ' version.config | sed -e 's!^version = !!'
```

Comment:
Is `sed` guaranteed to exist? Might be easier to use the `perl` shell for the run command because that will always work the same way on all platforms.

Reply:
Those images come preinstalled with lots of stuff you can check here: https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md

If sed wasn't available i would probably switch to powershell though